languages/js/src/Polar.ts

```diff
@@ -79,7 +79,7 @@ export class Polar<Query, Resource> {
     this.registerClass(Number, { name: 'Float' });
     this.registerClass(String);
     this.registerClass(Array, { name: 'List' });
-    this.registerClass(Object, { name: 'Dictionary' });
+    this.registerClass(Map, { name: 'Dictionary' });
```

Comment:
This is the "big" change. By making `Map` the default dictionary choice, we're ensure that all other classes get treated as external instances with an unknown type.